shop/shop/src/main/java/ro/msg/learning/shop/controller/OrderController.java

```diff
@@ -0,0 +1,21 @@
+package ro.msg.learning.shop.controller;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.shop.dto.OrderDto;
+import ro.msg.learning.shop.mapper.OrderMapper;
+import ro.msg.learning.shop.model.OrderE;
+import ro.msg.learning.shop.service.IOrderService;
+
+@RestController
+@RequestMapping("/orders")
+@RequiredArgsConstructor
+public class OrderController {
+    private final IOrderService service;
+    private final OrderMapper mapper;
+    @PostMapping("/order")
```

Comment:
No need to have POST on `/orders/order`. The default is post on `/orders` and you already have that from the `RequestMapping`
![image](https://user-images.githubusercontent.com/38504534/183649817-6a308743-3769-4513-aeb8-b0eccfbc7c67.png)